src/window.rs

```diff
@@ -628,6 +628,8 @@ impl Window {
     ///
     /// - **iOS:** Can only be called on the main thread.
     /// - **Android:** Will always return `None`.
+    /// - **Wayland** Can return `Borderless(None)` in some scenarios
+    ///
```

Comment:
```suggestion
    /// - **Wayland** Can return `Borderless(None)` in some scenarios.
    ///
```

I personally am not a huge fan of the "in some scenarios". Can we be more concrete here?